code/places/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val mavenPlacesVersion: String by project
```

Comment:
Why do we need this?

Reply:
This picks the value from gradle.properties

We dont need the mavenPlacesVersion variable in this gradle file. Removing them.
Thanks for pointing out